spec/modules/batch_spec.rb

```diff
@@ -70,6 +70,30 @@
       expect(BatchedSizeWorker).to have_enqueued_job([["bar"], ["bar"], ["bar"]])
       expect(batch.size).to eq(7)
     end
+
+    it 'must put all jobs at once to worker queue' do
+      batch = subject.new(BatchedAtOnceWorker.name, 'batched_at_once')
+
+      expect(batch.could_flush?).to be_falsy
+      7.times { BatchedAtOnceWorker.perform_async('bar') }
+      batch.flush
+      expect(BatchedAtOnceWorker).to have_enqueued_job([["bar"],["bar"],["bar"]])
+      expect(batch.size).to eq(1)
+    end
+
+    it 'must put all jobs at once to queue of time came and at once is set' do
+      batch = subject.new(BatchedAtOnceIntervalWorker.name, 'batched_at_once_interval')
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.